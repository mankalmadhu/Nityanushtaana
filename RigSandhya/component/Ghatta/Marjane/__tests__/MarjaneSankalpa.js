import React from 'react';
import ಮಾರ್ಜನೆಸ೦ಕಲ್ಪ from '../MarjaneSankalpa';
import Renderer from 'react-test-renderer';

test("ಮಾರ್ಜನೆಸ೦ಕಲ್ಪ", () => {
  const marjaneSankalpa = Renderer.create(
    <ಮಾರ್ಜನೆಸ೦ಕಲ್ಪ />
  ).toJSON();

  expect(marjaneSankalpa).toMatchSnapshot();

});