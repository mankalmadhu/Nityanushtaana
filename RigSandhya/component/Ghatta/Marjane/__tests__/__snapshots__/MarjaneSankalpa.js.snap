// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಮಾರ್ಜನೆಸ೦ಕಲ್ಪ 1`] = `
<View>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#3bdde5",
        "fontFamily": "Ganapati",
        "fontSize": 19,
        "lineHeight": 50,
        "marginLeft": 12,
        "textAlign": "justify",
      }
    }
  >
    ಮಾರ್ಜನೆ
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#6695e2",
        "fontFamily": "Ganapati",
        "fontSize": 18,
        "lineHeight": 50,
        "marginLeft": 12,
        "textAlign": "justify",
      }
    }
  >
    ಆಪೋಹಿಷ್ಠೇತಿ ತ್ರಿಚರ್ಚಸ್ಯ ಸೂಕ್ತಸ್ಯ, ಅ೦ಬರೀಶಃ ಸಿ೦ಧು ದ್ವೀಪ ಋಷಿಃ, ಆಪೋ ದೇವತಾ, ಗಾಯತ್ರೀ ಛ೦ದಃ ಮಾರ್ಜನೇ ವಿನಿಯೋಗಃ ॥
  </Text>
</View>
`;
